mm2src/coins/lp_coins.rs

```diff
@@ -252,6 +252,8 @@ use utxo::utxo_standard::{utxo_standard_coin_with_policy, UtxoStandardCoin};
 use utxo::UtxoActivationParams;
 use utxo::{BlockchainNetwork, GenerateTxError, UtxoFeeDetails, UtxoTx};
 
+mod nft;
```

Comment:
Please conditionalliy compile all the code that is directly related with nft integration using `enable-nft-integration` feature flag.

So we will avoid this in binaries and QA or devs can build mm2 using this feature to debug/check the nft integration.

Reply:
Done